bundles/org.openhab.automation.jythonscripting/README.md

```diff
@@ -1,4 +1,11 @@
-# Jython Scripting
+# Jython Scripting (DEPRECATED)
+
+::: tip Note:
+Currently, the development of Jython stopped at version 2.7 with no definite timeline to support Python 3.x.
+The 3rd party openHAB helper library for Jython is also no longer maintained.
+We would not recommend using Jython scripting at this point in time.
+For alternatives, check out the list of other supported [automation add-ons](/addons/).
```

Comment:
Is it a valid link ?
Does not work for me.

Reply:
I think it will work, but I've changed it to an absolute url just in case.